### DISCUSSION TITLE
TOTP (Authenticator app) Error while validating your second factor

### DISCUSSION THREAD

**Discussion by inhedgerow** [nextcloud/all-in-one#5245] [Questions] [answered]
### Steps to reproduce
1.Try to log in
2.Try Authenticator app
3.Try backup codes

### Expected behavior 
Login

### Actual behavior 
TOTP (Authenticator app)
Error while validating your second factor

**Comment by szaimen** [comment, marked as answer]
Hi, please open a thread for this at https://help.nextcloud.com

**Reply by inhedgerow** [reply]
Thanks. Was trying to do so but page was not responding. Thought also perhaps it was an aio specific problem. Will do so.
